projects/VitalRecord/VitalRecord_util.cs

```diff
@@ -417,57 +424,6 @@ public static void updateGivenHumanName(string[] value, List<HumanName> names, H
             }
         }
 
-        /// <summary>Helper method to validate that all PartialDate and PartialDateTime exensions are valid and have the valid required sub-extensions.</summary>
```

Comment:
could we keep this helper function for now and just don't call it anywhere? Add a comment that we are reevaluating the partial date handling with the standards team